go/obscuronode/enclave/enclave.go

```diff
@@ -115,7 +117,7 @@ func (e *enclaveImpl) start(block types.Block) {
 				if !found {
 					env.processedTxsMap[tx.Hash()] = tx
 					env.processedTxs = append(env.processedTxs, tx)
-					executeTx(env.state, tx, env.header)
+					evm.ExecuteTransactions([]nodecommon.L2Tx{tx}, env.state, env.header, e.storage, e.chainID)
```

Comment:
Yes, we'd want that. It would be the equivalent of the "Blockchain" object that geth has.
My thinking is that the "Storage" object can become that but in a future iteration. 
Baby steps :)